go/local/local.go

```diff
@@ -7,4 +7,4 @@ import (
 	_ "golang.org/x/text"
 )
 
-func main() {}
+func Local() {}
```

Comment:
This was unrelated. You can't import a main, so I fixed that. Didn't change anything.